test/e2e/usersignup_test.go

```diff
@@ -533,8 +534,13 @@ func (s *userSignupIntegrationTest) TestCapacityManagementWithManualApproval() {
 			Execute(s.T())
 		userSignup := user.UserSignup
 
+		_, spcNotReadyError := wait.For(t, hostAwait.Awaitility, &toolchainv1alpha1.SpaceProvisionerConfig{}).FirstThat(
+			assertions.Has(spaceprovisionerconfig.ReferenceToToolchainCluster(memberAwait1.ClusterName)),
+			assertions.Is(testSpc.NotReady()))
+
 		// then
 		s.userIsNotProvisioned(t, userSignup)
+		require.NoError(t, spcNotReadyError)
```

Comment:
Is there any reason for not checking the error right after calling the function?
```suggestion
			assertions.Is(testSpc.NotReady()))
			require.NoError(t, spcNotReadyError)

		// then
		s.userIsNotProvisioned(t, userSignup)
```

Reply:
The reason is just the visual when/then separation...

Reply:
ok, then at least swap it, it's fine to put it into the `// then` section 
```suggestion
			assertions.Is(testSpc.NotReady()))

		// then
		require.NoError(t, spcNotReadyError)
		s.userIsNotProvisioned(t, userSignup)
```
but not after calling `s.userIsNotProvisioned(t, userSignup)`. When I saw it, I was wondering how is the execution of `s.userIsNotProvisioned` related to the returned error :-)

Reply:
The new test looks a bit different so this is no longer applicable.

Reply:
uh, no, the above comment is completely wrong :)